src/openms/source/ANALYSIS/TOPDOWN/QScore.cpp

```diff
@@ -62,28 +62,28 @@ namespace OpenMS
 
   std::vector<double> QScore::toFeatureVector_(const PeakGroup *pg, const int abs_charge)
   {
-    std::vector<double> fvector(6);
+    std::vector<double> fvector(5);
```

Comment:
I was assuming the length of this vector corresponds to the weights vector. please add a comment in the code